config/locales/en.yml

```diff
@@ -479,44 +479,65 @@ en:
         one: "hour"
         other: "hours"
 
-  default_activity_management: "Management"
-  default_activity_design: "Design"
   default_activity_development: "Development"
+  default_activity_management: "Management"
+  default_activity_other: "Other"
+  default_activity_specification: "Specification"
+  default_activity_support: "Support"
   default_activity_testing: "Testing"
-  default_activity_Support: "Support"
-  default_activity_Other: "Other"
-  default_status_new: "new"
-  default_status_specified: "specified"
-  default_status_confirmed: "confirmed"
-  default_status_to_be_scheduled: "to be scheduled"
-  default_status_scheduled: "scheduled"
-  default_status_in_progress: "in progress"
-  default_status_tested: "tested"
-  default_status_on_hold: "on hold"
-  default_status_rejected: "rejected"
-  default_status_closed: "closed"
+  default_color_black: "Black"
+  default_color_blue: "Blue"
+  default_color_blue_dark: "Blue-dark"
+  default_color_blue_light: "Blue-light"
+  default_color_green_dark: "Green-dark"
+  default_color_green_light: "Green-light"
+  default_color_grey-dark: "Grey-dark"
+  default_color_grey-light: "Grey-light"
+  default_color_grey-medium: "Grey-medium"
```

Comment:
Is there a specific set of characters we are allowed to use?

If yes: we should tell @RobinWagner that he updates the crowdin docs
If no: Why don't you call it `light grey`, `medium grey`, etc, instead of this yoda-talk with dashes?